packages/terminal/src/browser/terminal-widget-impl.ts

```diff
@@ -60,6 +61,23 @@ export interface TerminalContribution {
     onCreate(term: TerminalWidgetImpl): void;
 }
 
+class TerminalBufferImpl implements TerminalBuffer {
```

Comment:
suggestion: Maybe move this and the interface to its own file. The files they are in are allready pretty long

Reply:
sure, makes sense